presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/JdbcMetadata.java

```diff
@@ -168,11 +169,19 @@ public void dropTable(ConnectorSession session, ConnectorTableHandle tableHandle
     @Override
     public ConnectorOutputTableHandle beginCreateTable(ConnectorSession session, ConnectorTableMetadata tableMetadata, Optional<ConnectorNewTableLayout> layout)
     {
-        JdbcOutputTableHandle handle = jdbcClient.beginCreateTable(tableMetadata);
+        JdbcOutputTableHandle handle = jdbcClient.beginCreateTable(tableMetadata, false);
         setRollback(() -> jdbcClient.rollbackCreateTable(handle));
         return handle;
     }
 
+    @Override
+    public void createTable(ConnectorSession session, ConnectorTableMetadata tableMetadata, boolean ignoreExisting)
+    {
+        JdbcOutputTableHandle handle = jdbcClient.beginCreateTable(tableMetadata, ignoreExisting);
```

Comment:
We don't need to reuse the code to create a temporary table and rename it. Instead, we should create the table directly. Let's add a new method to `JdbcClient` and extract a method for the `CREATE TABLE` SQL construction inside `BaseJdbcClient.beginWriteTable()`.